### PR TITLE
Make email marketing opt-in for new contacts

### PR DESCRIPTION
**Pull request by rachaelcodes**
## Description of change

When adding a new company contact, we now need to opt them into email updates from us (for data protection reasons) - this PR updates the new company form to reflect this. 

## Test instructions
Go to the companies list, and pick a company.
Click 'Company Contacts'
Click 'Add contact'
Scroll down the form, and the email checkbox should read: `The company contact does accept email marketing`
Complete the form and check the expected outcome is correct on the contact summary page.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: src/apps/contacts/services/form.js [hunk 1]
```diff
@@ -73,8 +73,7 @@ function saveContactForm(token, contactForm) {
         ['title', 'company', 'address_country']
       )
       const contactFormForApiRequest = merge({}, transformedContactForm, {
-        // database is positive on accepts, negative on rejects; UI is reverse, so flip that here
-        accepts_dit_email_marketing: !contactForm.rejects_dit_email_marketing,
+        accepts_dit_email_marketing: !!contactForm.accepts_dit_email_marketing,
```
Comment (LaurenQurashi) [inline src/apps/contacts/services/form.js:76]: Is this !! a typo or does it mean something?

Comment (rachaelcodes) [reply src/apps/contacts/services/form.js:76]: It means 'return true if this exists, or false if it doesn't' - a way to convert to a Boolean (and something I learned from another PR this week). There's a bit more info here: https://medium.com/better-programming/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054

Comment (LaurenQurashi) [reply src/apps/contacts/services/form.js:76]: Oh how fancy! 🎉 thanks Rachael 😄

Comment (adamledwards) [reply src/apps/contacts/services/form.js:76]: @LaurenQurashi you can also use Boolean(value) to convert. 
My preferred method :) 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean

**Comment by adamledwards** [comment]
Visual test broken again :(

File: src/apps/contacts/services/form.js (hunk 1 above)
Comment (rachaelcodes) [reply src/apps/contacts/services/form.js:76]: Ah nice - that's much clearer!

**Pull request merged**